.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,5 @@
 <!-- Thanks for submitting a PR! Please before merging this PR ensure all the following steps are fullfilled: -->
 
 - [ ] Explain the **motivation** for making this change.
-- [ ] Provide a **test plan** demonstrating that the code is solid.
-- [ ] Match the **code formatting** of the rest of the codebase.
-- [ ] Two other colleagues must **approve** this change.
-- [ ] **QA** have tested the solution provided.
-- [ ] **Design and UX** must validate this change.
+- [ ] Provide a solid **test plan** for your changes.
+- [ ]  Link to the issue - if any [issue number here]()
```

Comment:
Maybe it is worth leaving the `Match the **code formatting** of the rest of the codebase.` part?

Reply:
We discussed it with @Gotusso in the Notion page, we already enforce it with linting/tests and comment, it's not needed in this case